dev/date-picker.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Please rebase the PR as there is now dev page on master.

Reply:
Done